packages/db/index.js

```diff
@@ -1295,6 +1295,30 @@ export class DBClient {
     }
   }
 
+  /**
+   * @param {string} userId
+   * @param {string} agreement
```

Comment:
Will you add a `@returns` ?

Reply:
https://github.com/web3-storage/web3.storage/pull/2028/commits/7631d7a57bb35237590b3c8596d689a81367d4f3